lib/mock-axios.ts

```diff
@@ -258,6 +258,16 @@ MockAxios.getReqByMatchUrl = (url: RegExp) => {
     return _findReqByPredicate((x) => url.test(x.url));
 };
 
+MockAxios.getReqByMatch = (opts: object) => {
+    return _findReqByPredicate((x) => {
```

Comment:
I personally prefer a more concise way of writing this, so maybe this:

```ts
return _findReqByPredicate(x => Object.entries(opts).every(([key, value]) => value.test(JSON.stringify(x[key]))));
```

is better?

Reply:
Yes! `Object.entries()` for the win :)

Reply:
Done!